modules/virt-adding-public-key-cli.adoc

```diff
@@ -0,0 +1,198 @@
+// Module included in the following assemblies:
```

Comment:
Hey, what is this doc related to ?

Reply:
Oh that has been added mistakenly, due to rebasing issues. I have removed it.

Reply:
Do not see this to be resolved, i see the file again, could you please help check ? thanks !!